src/components/App.jsx

```diff
@@ -0,0 +1,24 @@
+
+import { Routes, Route } from "react-router-dom"
+import { useState } from "react"
+import Header from './Header'
+import Home from './Home'
+import Articles from './Articles'
+import '../App.css'
+
+function App() {
+
+  const [articles, setArticles] = useState([])
```

Comment:
Does this state need to be declared all the way at the top of tree like this? Perhaps you do need to for further down the line but just double check that this would be the case